commit 7b3b2c1bf9b4d9a68333beaf76a2609518b636d6 (refs/changes/69/2869/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-25T13:26:12-07:00 (1 year, 11 months ago)
    
    TOOLS-1898 sdcnode updates for Node.js Oct 2017 vulns
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/Makefile b/Makefile
index 96706d3..075d602 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ CLEAN_FILES	+= node_modules cscope.files
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v4.8.5
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG=zone
